### DISCUSSION TITLE
Aufgabenstellung Assignment 1 Aufgabe 4.1

### DISCUSSION THREAD

**Discussion by juwess** [irgroup-classrooms/dis08-2021#30] [Q&A] [answered]
Hallo,
einige Kommilitonen und ich sind sich unsicher wie genau die erste Aufgabe der Exercise 4 zu verstehen ist.
Soll erst angegeben werden wie viele Zeilen allgemein pro csv und in total vorhanden sind und dann anschließend wie oft trihard oder kappa zusammengefasst in allen csv in total vorkommen oder soll einmal angegeben werden in wie vielen Zeilen trihard oder kappa vorkommen und dann nochmal wie viele total occurences in allen csv Dateien zusammen vorhanden sind? Oder haben wir die Aufgabe gänzlich falsch verstanden?

Julian Weß (Data and Information Science)

**Comment by HaakFab** [comment, marked as answer]
Guten Tag, 
In einer anderen Discussion habe ich diese Frage soeben hoffentlich klären können:

Finden Sie zunächst für alle .csv Dateien die Anzahl der Zeilen, in denen mindestens einer der beiden Terme "kappa" oder "trihard" (keine weiteren Zeichen) vorkommt. In einem Weiteren Schritt (Command), lassen Sie sich die Gesamtanzahl aller Vorkommen ausgeben. Diese sollte also die Summe der Counts aus dem ersten Command entsprechen.
Ich habe die Formulierung entsprechend angepasst!
Hinweis: Auch 4.2 wurde leicht angepasst, hier ist das "and in total" weggefallen.

Ich hoffe, es ist nun deutlicher, was Sie tun sollen.
VG,
Fabian Haak

**Reply by juwess** [reply]
Vielen Dank, das hat es aufgeklärt!
Dann habe ich noch eine abschließende Frage: Sollen in der "Lösungsdatei" exercise04.sh nur die Commands der Aufgaben abgebildet werden oder auch die Lösungen?
Beste Grüße 
Julian Weß

**Comment by HaakFab** [comment]
Guten Morgen,
exercise04.sh enthält nur Ihre commands, wie in der Aufgabenstellung gefordert. Ihre Dokumentation bestehend aus Commands, ihren Anmerkungen und Erläuterungen sowie den produzierten Ausgabe kommt in die Markdown-formatierte Datei "exercise04.md". 
VG,
Fabian Haak
